app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
+ @import 'bootstrap';
```

Comment:
You're going to see this a lot. Please make sure you have settings in your IDE to add a new line at the end of every file.